src/components/NoticeText/CampaignRegistration10DlcNoticeText.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+const CampaignRegistration10DlcNoticeText: React.FC = () => (
+  <>
+    <p>
+      <b>
+        You have started but not yet completed your 10DLC registration form.
+      </b>
```

Comment:
`<b>` elements now have a specific semantic meaning that is used to draw attention to portions of text without giving importance or emphasis. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

In this particular case, I would replace with `<strong>` or apply CSS styling on the paragraph.

See also:
- [A good StackOverflow post on the topic](https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em)
- [A good blog post on the topic](http://stellify.net/html5-b-and-i-tags-are-going-to-be-useful-read-semantic-again/)
- [The MDN docs on the `<strong>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong)

Reply:
I am so behind the times on this sort of stuff 😭